spec/presenters/two_factor_auth_code/totpable.spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+describe TwoFactorAuthCode::Totpable do
+  describe '#auth_app_fallback_link' do
+    context 'when totp option is not enabled' do
+      let(:presenter) do
+        TwoFactorAuthCode::PhoneDeliveryPresenter.new(attributes_for(:generic_otp_presenter))
+      end
+
+      it 'returns a single period to properly format fallback options' do
+        expect(presenter.auth_app_fallback_link).to eq('.')
+      end
+    end
+
+    context 'when totp option is enabled' do
+      let(:presenter) do
+        data = attributes_for(:generic_otp_presenter).merge!(totp_enabled: true)
+        TwoFactorAuthCode::PhoneDeliveryPresenter.new(data)
+      end
+
+      it 'returns a link to use an authenticator app' do
+        actual = presenter.auth_app_fallback_link
+        # rubocop:disable Style/StringLiterals
```

Comment:
When disabling a cop in this way, it must always be enabled after the point where you no longer want to disable it. However, it's even better to fix the offense. In this case, it's easy. Change line 24 to this:
```ruby
link = '<a href="/login/two_factor/authenticator">authentication app</a>'
```